PHP/BestPractices.md

```diff
@@ -3,13 +3,19 @@
 # Junior Level
 [PSR-1 basic coding standard](http://www.php-fig.org/psr/psr-1/)
 
-[PSR-2 coding style guide](http://www.php-fig.org/psr/psr-2/)
+[~~PSR-2 coding style guide~~](http://www.php-fig.org/psr/psr-2/) 
```

Comment:
Maybe we need specify that PSR-2 is deprecated and PSR-12 is recommended

Reply:
Yeah, I agree. I added the mark in another place and will replicate here too